includes/ApplHandlingByStr.php

```diff
@@ -74,6 +74,10 @@ function showFullStatus( $stat_str )
 		{
 			echo "<td><b>FORWARDED</b> from ".$str[0]."</td>";
 		}
+		else if( $str[1] == 'G' )
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Usage of ELSE IF is discouraged; use ELSEIF instead
* Expected 1 space after IF keyword; 0 found
* Expected 0 spaces after opening bracket; 1 found
* Expected 0 spaces before closing bracket; 1 found
* Expected 1 space after closing parenthesis; found 9

<!-- Comment by [SideCI](https://sideci.com) -->